python/python/glide/async_commands/core.py

```diff
@@ -1527,6 +1527,26 @@ async def sismember(
             await self._execute_command(RequestType.SIsMember, [key, member]),
         )
 
+    async def sinter(self, keys: List[str]) -> Set[str]:
+        """
+        Gets the intersection of all the given sets.
+        See https://redis.io/docs/latest/commands/sinter/ for more details.
+
+        Args:
+            keys (List[str]): The keys of the sets.
+
+        Returns:
+            Set[str]: A set of members which are present in all given sets.
+                Missing or empty input sets cause an empty set response.
+
+        Examples:
+            >>> await client.sadd("my_set1", ["member1", "member2"])
```

Comment:
nit: you don't need to include other commands, just describe what's in the Set via a comment.

Reply:
I just follow other examples.